data/avgi/avgi 1 first contact.txt

```diff
@@ -0,0 +1,834 @@
+		system "Ensemble"
+		fleet
+			names "aberrant names"
+			variant
+				"Aberrant Latte" 4
+				"Aberrant Chomper" 7
+				"Aberrant Pileup" 4
+		fleet
+			names "aberrant names"
+			variant
+				"Aberrant Latte" 7
+				"Aberrant Chomper" 4
+				"Aberrant Whiskers" 4
+		fleet
+			names "aberrant names"
+			variant
+				"Aberrant Traincar" 5
+				"Aberrant Mole" 3
+		fleet
+			names "aberrant names"
+			variant
+				"Aberrant Trip" 3
+				"Aberrant Triplet (Disable-able)" 2
+	on complete
+		conversation
+			`You land back on Weledos after the battle, while most of the surviving Avgi ships begin temporary repairs in orbit before they travel back to Napigeo Station. While you and Darius are checking the <ship> for damage, you hear someone walk up with distinctly human footsteps.`
+			`	"So, those rumors weren't all talk. There really is another one of us here." You look up to see a woman in combat fatigues standing in front of you, hand extended.`
+			choice
+				`	(Shake her hand.)`
+					goto handshake
+				`	"Who are you?"`
+			`	She drops her hand after a moment. "Name's Sora. And you?"`
+				goto name
+			label handshake
+			`	"Name's Sora. And you?"`
+			label name
+			`	"Captain <first> <last>," you say.`
+			`	"And I see you've got a tagalong, too. Finally got ol' Darius to come out of his shell, did you?" You look over at Darius, a look of deep shame on his face as he stares at the ground.`
+			`	"Now, I'm very interested in how exactly you managed to find your way here. Care to explain?" she asks.`
+			`	You fill Sora in on how your travels brought you here, and how you were able to acquire a jump drive. Darius says nothing throughout, standing by silently.`
+			`	"Amazing. So, your jump drive can get us out of here then?"`
+			choice
+				`	"I'm afraid not. It can't return to human space for some reason."`
+				`	"No, but I'm not giving up on finding a different way out."`
+				`	"Nope!"`
+			`	"Damn. That would be too easy, I guess. But it's been too long since I've seen a fresh human face. I've been working for the Twilight Guard here for months, trying to build up a second line of defenses in case that station in Arche falls. At least today's battle shows those defenses, together with a supporting fleet, can hold. You mind if I tag along with you, in case you do manage to find a way out?"`
+			choice
+				`	"Of course."`
+					goto yes
+				`	"Sure, as long as Darius doesn't mind."`
+			`	You look over at Darius, who responds after a few seconds of silence. "Of course you can come along, Sora."`
+			label yes
+			`	"Sounds good to me, then. I'll see you around when you're ready to leave." She walks off to collect her belongings.`
+			`	A few hours later, an Avgi cruiser, a Refraktos-class Laserstar, settles down on the pad next to yours, and an Avgi officer disembarks.`
+			`	"Salutations. Hello, Captain <first> <last>. My name is Kokkino, Captain of the Distant Collision. Tell me, is the <ship> a warship, question?"`
```

Comment:
Probably not worth it change things to specify the Distant Collision is around as part of the fleet during the battle just for the consistency, but the thought came to mind.